sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/policy/RequestRetryOptions.java

```diff
@@ -113,8 +147,17 @@ public int getMaxTries() {
 
     /**
      * @return the maximum time, in seconds, allowed for a request until it is considered timed out.
+     * @deprecated Please use {@link RequestRetryOptions#getTryTimeoutDuration()}
      */
+    @Deprecated
     public int getTryTimeout() {
+        return (int) this.tryTimeout.getSeconds();
+    }
+
+    /**
+     * @return the maximum time, in seconds, allowed for a request until it is considered timed out.
+     */
+    public Duration getTryTimeoutDuration() {
```

Comment:
Any better ideas for this? :(

Reply:
I think this is consistent with what we did for like blockSizeLong, so I think it's fine.